lib/rubocop/cop/style/case_equality.rb

```diff
@@ -4,6 +4,18 @@ module RuboCop
   module Cop
     module Style
       # This cop checks for uses of the case equality operator(===).
+      #
+      # @example
+      #   # bad
+      #   Array === something
+      #   (1..100) === 7
+      #   /something/ === some_string
```

Comment:
I think we should add an example of `==` case.
For example:

```ruby
# bad
some_string === 'something'

# good
some_string == 'something'
```

WDYT?

Reply:
I think we should not mix Equality and Case Equality in this example.
https://stackoverflow.com/questions/3422223/vs-in-ruby

maybe it is better to separate the example in three parts.
I could update the PR this Weekend like this:

```ruby
@example
 # Class of something
 # bad
 Array === something

 # good
 something.is_a?(Array)

@example
 # Inclusion in Range
 # bad
 (1..100) === 7

 # good
 (1..100).include?(7)
 
@example
 # Regular expression match
 # bad
 /something/ === some_string

 # good
 some_string =~ /something/
 ```